axon-server/security/access-control.md

```diff
@@ -2,86 +2,146 @@
+AxonIQ-Access-Token: my-token-value-here
 ```
 
-The token that you must use here is the token you have set in the `axonserver.properties` file. In this case you will be prompted for a password \(you can pass the password in the command line using the -p option\). Valid roles are READ and ADMIN.‌
+### **Users - Axon Server SE UI Console**
+
+You can create users either through the command line or through the dashboard web application. The _**initial user**_ has to be created through the command line as you would not be able to login tp the dashboard. 
+
+To do this execute the _"register-user"_ command,
+
+```text
+$ java -jar axonserver-cli.jar register-user -u username -r roles [-p password] -S http://[node]:[port] [-t token]
+```
+
+_Mandatory parameters_
+
+* _**-u**_ refers to the username.
+* _**-r**_ refers to the role of the user. Specify multiple roles by giving a comma separated list \(without spaces\) -&gt; READ,ADMIN. 
+* _**-p**_ refers to the password of the user. If you do not specify a password with the -p option, the command line interface will prompt you for one.‌
+
+_Optional parameters_
+
+* _**-S**_ refers to the server to send the command to and if not supplied connects by default to [http://localhost:8024](http://localhost:8024). For Axon Server SE, the URL for the Axon Server SE will be the single running node
+* _**-t**_  refers to the access token to authenticate at the server to which the command is sent to. This should be the same as the token setup in the `axonserver.properties` file.
+
+Users can also be added using the REST API / UI Console that Axon Server SE provides. The CLI also allows the capabilities to [list all users](../administration/admin-configuration/command-line-interface.md#users) as well as[ delete specific users](../administration/admin-configuration/command-line-interface.md#users).
 
 ## Axon Server Enterprise Edition <a id="axon-server-enterprise-edition"></a>
 
-Axon Server Enterprise Edition has a much more fine-grained access control mechanism. Here you can grant specific roles to application that will allow specific operations. In this edition you can also grant more specific roles to users. Apart from just assigning the roles, you must also indicate for which context the role is granted, so that an application that has rights on only one context is not able to access data from other contexts.‌
+Axon Server Enterprise Edition provides a fine-grained access control mechanism for
 
-Instead of setting a single token in `axonserver.properties`, you must now register applications with specific roles. Since every request requires a token or a valid user, there is a special way of creating the first application and/or user. You can create those using the command line interface on an Axon Server node, from the same directory where Axon Server was started.‌
+* Client Applications connecting to Axon Server SE.
```

Comment:
Shouldn't this state `Axon Server EE` instead of `Axon Server SE`?

And, doesn't it also include access control between Axon Server instances?

Reply:
Ah, I see it's described under header `### Axon Server cluster`